cmd/precheck/precheck.go

```diff
@@ -85,6 +106,10 @@ func AllPrechecks() {
 		wg.Done()
 
 	}()
+	go func() {
+		generateEnvrc()
+		wg.Done()
+	}()
 	go func() {
```

Comment:
just noticed, what is this go routine for? Can we remove it please?

Reply:
ya previously  it was the docker thing to check enable store